src/com/theironyard/novauc/ATM.java

```diff
@@ -0,0 +1,111 @@
+                "                 |_     ____||        ||        ||    ____||     ____|\n" +
+                "                   |   |.———.|   ||   ||   ||   ||   '————. |   |.———.\n" +
+                "                   |   ||   ||   ||   ||    ____||    ____| |   ||   |\n" +
+                "                   |   ||   ||   ||   ||   |.———.|   |      |   ||   |\n" +
+                "                   |        ||   ||   ||        ||   |      |        |\n" +
+                "                   `.______.´|___||___|`.______.´|___|      `.______.´\n" +
+                "         .´¯¯¯¯¯¯`.|¯¯¯||¯¯¯|-¯¯¯'————..´¯¯¯¯¯¯|¯¯¯¯¯¯¯¯|     |¯¯¯¯¯¯|\n" +
+                "         |___'|   ||   ||   ||    ____||   ..   ¯\\    ´¯       ¯`  /¯\n" +
+                "         .————'   ||   ||   ||   |.———.|   ||   | \\    \\        / /\n" +
+                "         |   ..   ||   ||   ||   ||   ||   ||   |  \\    \\      / /\n" +
+                "         |   ''   ||   ''   ||   ''   ||   ''   |   \\    \\    / /\n" +
+                "         `._______|`.______.´`.______.´`.______.´    \\    \\  / /\n" +
+                "                                                      \\    \\/ /\n" +
+                "           Bank of America support                     \\     /\n" +
+                "                                                        \\   /\n" +
+                "                      " +
+                "" +
+
+                "Get your lube ready, " + name + " HAHAHAHA!");
+        System.out.println("Enter your damn pin! [pin/leave]");
+        String pin = scanner.nextLine();
+
+
+        if (pin.equalsIgnoreCase("1234")) {
+            System.out.println("Here are your insufficient funds...do better.");
+
+        } else if (pin.equalsIgnoreCase("leave")) {
+            System.out.println("Bye Felicia!");
+
+        } else {
+            throw new Exception("Stand there in the cold then, dummy.");
+        }
+
+
+
+            System.out.println("Check your [balance]");
+            String  j = scanner.nextLine();
+
+
+            if (j.equalsIgnoreCase("balance"))
+            {System.out.print(" Your balance is" + customer.balance);
+            }
+
+
+            System.out.println("Check your [Checking/Savings/Deposit/Withdraw/Cancel]");
+            String prompt = scanner.nextLine();
+
+            if (prompt.equalsIgnoreCase("Checking")) {
+                System.out.println("Your checking is "+customer.checking);
+            } else if (prompt.equalsIgnoreCase("Savings")) {
+                System.out.println("Your savings balance is " + customer.savings);
+            }
+            else if (prompt.equalsIgnoreCase("Deposit")) {
+                System.out.println("Your balance is" + customer.balance);
+            }
+
+
+            else if (prompt.equalsIgnoreCase("Withdraw")) {
+                System.out.println(+customer.balance - customer.savings);
+            }
```

Comment:
After they select withdraw.  you need to add lines of code that ask how much they want to withdraw.  Then take in a value of how much to withdraw.  Then subtract that amount from their account.  Then tell them to take their cash and let them know what their new balance is.  Add this code and then resubmit.